src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Instead of empty tag <div> for logo it is better to use tag <img> with attributes 'src' and 'alt'.
Also, to improve reading, use more understandable classes, not header__logo, but header__logo-image.